<launch>
  <arg name="cloud_topic" default="/cloud_registered"/>
  <arg name="buffer_duration" default="3.0"/>
  <arg name="publish_rate" default="1.0"/>

  <!-- <node pkg="trunk_detection" type="cloud_accumulator" name="cloud_accumulator" output="screen">
    <param name="cloud_topic" value="$(arg cloud_topic)"/>
    <param name="buffer_duration" value="$(arg buffer_duration)"/>
    <param name="publish_rate" value="$(arg publish_rate)"/>
  </node> -->

  <node pkg="trunk_detection" type="cloud_processing" name="cloud_processing" output="screen"/>
  
  <node pkg="trunk_detection" type="cloud_cropped" name="cloud_cropped" output="screen"/>

  <node pkg="trunk_detection" type="fit_cyl" name="fit_cyl" output="screen"/>

</launch>
